BigData_updateAgilityPressure.R

```diff
@@ -0,0 +1,44 @@
+### Load libraries
+
+library(tidyverse)
+library(brms)
+library(tidybayes)
+library(lme4)
+library(dplyr)
+library(rlang)
+library(reshape2)
+
+rm(list=ls())
+
+### Import Parent Data
+
+ParentDat <- read.csv('C:/Users/eric.honert/Boa Technology Inc/PFL Team - General/BigData/DB_V2/CyclingPowerDB_V2.csv',nrows=1)
```

Comment:
Pls update for the agility db.
Should be: 'C:/Users/kate.harrison/Boa Technology Inc/PFL Team - General/BigData/DB_V2/AgilityPressureDB.csv'